Dev playback speed

Briggs-Richardson opened the pull request:
DO NOT MERGE

Briggs-Richardson committed:
Added a custom combobox with playback speeds, connected to a Controller function which updates the playback speed upon change

Briggs-Richardson committed:
Adds controls for playback speed

This patch adds a playback speed combobox located in the media control panel,
where the user can select playback speeds of 0.25, 0.5, 0.75, 1.0, 1.25, 1.5,
1.75, and 2.0. The combobox itself is a custom combobox in order to print a
label in the combobox text.

The playback speed is not recognized if it is changed before the video is
loaded and played. This "bug" may be fixed with the addition of the play and
pause button. If not, it will need to be addressed in the future.

Testing Steps
  1. Run main.py to start the application.
  2. Load a video and play it.
  3. Verify that the playback speed changes accurately according each option in
     the playback speed combobox.

Type: New Feature

Jcobo314 commented on View/media_control_panel.py:
```diff
@@ -15,9 +15,13 @@ def __init__(self):
         horizontal_layout = QHBoxLayout()
 
+        # Create a playback speed combobox as a media control widget.
         self.playback_speed_combobox = PlaybackSpeedComboBox()
```
combo box is 2 words so just add an underscore inbetween

Briggs-Richardson committed:
Adds controls for playback speed

This patch adds a playback speed combobox located in the media control panel,
where the user can select playback speeds of 0.25, 0.5, 0.75, 1.0, 1.25, 1.5,
1.75, and 2.0. The combobox itself is a custom combobox in order to print a
label in the combobox text.

The playback speed is not recognized if it is changed before the video is
loaded and played. This "bug" may be fixed with the addition of the play and
pause button. If not, it will need to be addressed in the future.

Testing Steps
  1. Run main.py to start the application.
  2. Load a video and play it.
  3. Verify that the playback speed changes accurately according each option in
     the playback speed combobox.

Type: New Feature
